### PR TITLE
feat(i18n): translate title

### PR DESCRIPTION
**Pull request by Build774**
## Description

Translated titles.  

## How Has This Been Tested?

Manually tested.

## Notes

I'm not sure it's ok to place titles into `titles.json`.

Though `Music` is translated as `音楽` in the sidebar, I used `楽曲` for titles because `楽曲一覧` is more natural than `音楽一覧`.  
In addition, `Music Select` is translated as `楽曲選択` in the game.

### CODE REVIEW

**Comment by Build774** [comment]
GitHub shows `build (12.x) Expected — Waiting for status to be reported`...
Maybe I forgot to change something in [4174d0](https://github.com/Sekai-World/sekai-viewer/commit/4174d0cccae04e11d0dc4b15205c1b4d0a9595e2) ?

File: public/locales/ru/card.json [hunk 1]
```diff
@@ -15,7 +15,10 @@
   "sideStory": "Побочная история",
   "sideStory1Unlocked": "Побочная история 1 открыта",
   "sideStory2Unlocked": "Побочная история 2 открыта",
-  "sideStory_plural": "Побочные истории",
+  "sideStory_0": "",
+  "sideStory_1": "",
+  "sideStory_2": "",
```
Comment (Build774) [inline public/locales/ru/card.json:20]: I don't know why this is added, but even if I remove this, doing `npm run exti18n` will add it again.

File: public/locales/ru/common.json [hunk 2]
```diff
@@ -53,6 +53,9 @@
   "campaign": "Компания",
   "bug": "Баги",
   "update": "Обновления",
-  "character_plural": "Персонажи",
-  "level": "Уровень"
+  "character_0": "",
+  "character_1": "",
+  "character_2": "",
```
Comment (Build774) [inline public/locales/ru/common.json:58]: I don't know why this is added, but even if I remove this, doing `npm run exti18n` will add it again.

Comment (dnaroma) [reply public/locales/ru/common.json:58]: Russian seems to have different expressions for amount 0,1,2 of things, just leave it there, everything's fine

**Comment by dnaroma** [comment]
> GitHub shows `build (12.x) Expected — Waiting for status to be reported`...
> Maybe I forgot to change something in [4174d0](https://github.com/Sekai-World/sekai-viewer/commit/4174d0cccae04e11d0dc4b15205c1b4d0a9595e2) ?

Idk, but build(12.x) should not exist anymore

**Comment by Build774** [comment]
Maybe [this](https://github.community/t/build-12-x-expected-waiting-for-status-to-be-reported/17576/2)?
https://github.com/Sekai-World/sekai-viewer/settings/branches

**Comment by dnaroma** [comment]
> Though `Music` is translated as `音楽` in the sidebar, I used `楽曲` for titles because `楽曲一覧` is more natural than `音楽一覧`.
> In addition, `Music Select` is translated as `楽曲選択` in the game.

I agree with you. the sidebar `Music` can also be translated to `楽曲`

**Comment by Build774** [comment]
> the sidebar `Music` can also be translated to `楽曲`

Can I commit changes directly (i.e., without changing transifex)?

**Comment by dnaroma** [comment]
> > the sidebar `Music` can also be translated to `楽曲`
> 
> Can I commit changes directly (i.e., without changing transifex)?

Yes, I will upload it to transifex later

**Comment by Build774** [comment]
Ok. I'll do it soon.  
BTW may I also change `summary` to `概要` from `まとめ` in GachaDetail page?
I think it's a bit weird.

**Comment by dnaroma** [comment]
Yes you may change the translation that's not suitable

**Pull request merged**